app/workers/audio_callback_worker.rb

```diff
@@ -41,6 +42,9 @@ def perform(_sqs_msg, job)
 
     Shoryuken.logger.info("Updating #{job['type']}[#{audio.id}]: status => #{audio.status}")
     audio.save!
+    # announce the audio changes using the story as resource.
+    announce(:audio, :update, id: audio.story.id, resource: audio.story)
```

Comment:
Actually -- will having the subject here be `:audio` end up trapping us in an infinite loop of the audio announcement triggering the lambda triggering the audio announcement etc?